views/drones/create-form.hbs

```diff
@@ -1 +1,16 @@
-<h2>Create a new drone</h2>
\ No newline at end of file
+<h2>Create a new drone</h2>
+
+<form action="/drones/create" method="POST">
+<label> Name:
+    <input type="text" name="name">
+</label>
+<label> Propellers:
+    <input type="text" name="propellers">
+</label>
+<label> Speed:
+    <input type="text" name="maxSpeed">
```

Comment:
Try to add in the form the same validations that you use in the model schema. Ex: `max="18"`